curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md

```diff
@@ -41,11 +41,11 @@ Add a horizontal motion to the <code>div</code> animation. Using the <code>left<
 ```yml
 tests:
   - text: The <code>@keyframes</code> rule for <code>0%</code> should use the <code>left</code> offset of 0px.
-    testString: assert(code.match(/0%\s*?{\s*?background-color:\s*?blue;\s*?top:\s*?0(px)?;\s*?left:\s*?0(px)?;\s*?}/gi));
+    testString: assert(code.match(/0%\s*?{\s*?background-color:\s*?blue;\s*?top:\s*?0(px)?;\s*?left:\s*?0(px)?;\s*?}/gi) || code.match(/0%\s*?{\s*?background-color:\s*?blue;\s*?left:\s*?0(px)?;\s*?top:\s*?0(px)?;\s*?}/gi) || code.match(/0%\s*?{\s*?left:\s*?0(px)?;\s*?background-color:\s*?blue;\s*?top:\s*?0(px)?;\s*?}/gi));
```

Comment:
I have tested the below regex that is much simpler.  The extra `[^50]` added before the `0%` was to make sure the `50` and the `100` were not caught in the match.  Please click the `Commit suggestion` if you agree.
```suggestion
    testString: assert(code.match(/[^50]0%\s*?{[\s\S]*?left:\s*?0px(;[\s\S]*?|\s*?)}/gi));
```